www/js/components/add-alert-details.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-disable no-unused-vars*/
+import React, { Component } from 'react';
+
+import { RaisedButton, TextField } from 'material-ui';
+import DropDown from './drop-down';
+/*eslint-disable no-unused-vars*/
+
+export class AddAlertDetails extends Component {
+
+  componentDidMount() {
+    const {setDrawer} = this.props;
+    setDrawer( 'Enter Information' ) ;
+  }
+
+  onNameUpdate( event ) {
+    const {setPointName} = this.props;
+    const name = event.target.value;
+    setPointName( name );
+  }
+
+  onDescriptionUpdate( event ) {
+    const {setPointDescription} = this.props;
+    const description = event.target.value;
+    setPointDescription( description );
+  }
+
+  onPhotoAdd() {
```

Comment:
We'll discuss this thursday, but we should highlight this as being optional (as with other fields)